.github/workflows/build-and-test-arm.yml

```diff
@@ -84,3 +91,27 @@ jobs:
             echo "One or more matrix jobs failed."
             false
           fi
+
+  flakytests-generate-issues:
```

Comment:
Should these have unique names by arch?

Reply:
I'm not sure if it's needed, but doesn't hurt to change :)